File: src/main/java/cat/xlagunas/andrtc/token/TokenService.java
package cat.xlagunas.andrtc.token;

import java.util.List;

public interface TokenService {

    void addToken(Token token) throws ExistingTokenException;

    void removeToke(Token token);

    List<String> getUsersToken(long userId);

    List<String> getUsersToken(List<Long> userIdList);
}

File: src/main/java/cat/xlagunas/andrtc/token/TokenService.kt
package cat.xlagunas.andrtc.token

interface TokenService {

    @Throws(ExistingTokenException::class)
    fun addToken(token: Token)

    fun removeToke(token: Token)

    fun getUsersToken(userId: Long): List<String>

    fun getUsersToken(userIdList: List<Long>): List<String>
}
